src/SFA.DAS.EmployerApprenticeshipsService.Application/Queries/GetSentTransferConnectionInvitationQuery/SentTransferConnectionInvitationViewModel.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System.ComponentModel.DataAnnotations;
+using SFA.DAS.EAS.Domain.Models.TransferConnection;
+
+namespace SFA.DAS.EAS.Application.Queries.GetSentTransferConnectionInvitationQuery
+{
+    public class SentTransferConnectionInvitationViewModel
+    {
+        [Required]
+        [Range(1, 2)]
+        public int Choice { get; set; }
```

Comment:
Is there a way to make choice more descriptive as other a developer has to trawl through the code to find out what each choice means?